src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -1262,6 +1262,9 @@ struct alignas(4) HardwareTickConfig {
 /**
  * @brief Heading bias horizontal/vertical configuration settings.
  * @ingroup config_and_ctrl_messages
+ * @note
+ * Both HeadingBias values must be set for the system to use them.
+ * If one value is NOT set, the system will just output Invalid.
```

Comment:
`@ref SolutionType::Invalid`

Reply:
I'm not sure this is super useful actually. There's not much use for always putting out invalid -- we may want to just make the system not output anything at all if biases are not specified.